### ISSUE TITLE
fixed vulnerabilities reported

### ISSUE THREAD

**Issue by petersutty** [wazuh/wazuh#2891]

**Comment by petersutty** [comment]
debian 9.8 stretch
wazuh-manger 3.8.2-1
because of wrong oval files from debian (severity unknow) all the packages are reported because of rule 23504 - u have issue opened for this already
But I believe this should not be reported/checked at all if the package is on a version, where the vulnerability was fixed.

eg. 
Wazuh Notification.
2019 Mar 22 11:22:39

Received From: spossec02->vulnerability-detector
Rule: 23504 fired (level 7) -> "CVE-2005-2496"
Portion of the log(s):

{"vulnerability":{"cve":"CVE-2005-2496","title":"CVE-2005-2496","severity":"Unknown","published":"2005-09-05","state":"Pending confirmation","package":{"name":"ntp","version":"1:4.2.8p10+dfsg-3+deb9u2"},"reference":"http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2005-2496"}}
vulnerability.cve: CVE-2005-2496
vulnerability.title: CVE-2005-2496
vulnerability.severity: Unknown
vulnerability.published: 2005-09-05
vulnerability.state: Pending confirmation
vulnerability.package.name: ntp
vulnerability.package.version: 1:4.2.8p10+dfsg-3+deb9u2
vulnerability.reference: http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2005-2496

dpkg -l | grep ntp
ii  ntp                            1:4.2.8p10+dfsg-3+deb9u2       amd64        Network Time Protocol daemon and utility programs

CVE-2005-2496 is fixed in stretch	1:4.2.8p10+dfsg-3+deb9u2
https://security-tracker.debian.org/tracker/CVE-2005-2496

**Comment by TJOSERAFAEL** [comment]
Hi @petersutty ,

I have been able to reproduce the problem on a Debian 9.8 Stretch VM with Wazuh version 3.8.2:

```
** Alert 1557932469.6466: - vulnerability-detector,gdpr_IV_35.7.d,
2019 May 15 15:01:09 stretch->vulnerability-detector
Rule: 23504 (level 7) -> 'CVE-2005-2496'
{"vulnerability":{"cve":"CVE-2005-2496","title":"CVE-2005-2496","severity":"Unknown","published":"2005-09-05","state":"Pending confirmation","package":{"name":"ntp","version":"1:4.2.8p10+dfsg-3+deb9u2"},"reference":"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2005-2496"}}
vulnerability.cve: CVE-2005-2496
vulnerability.title: CVE-2005-2496
vulnerability.severity: Unknown
vulnerability.published: 2005-09-05
vulnerability.state: Pending confirmation
vulnerability.package.name: ntp
vulnerability.package.version: 1:4.2.8p10+dfsg-3+deb9u2
vulnerability.reference: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2005-2496
```

It has been fixed in Wazuh 3.9:

Debug output of `ossec-modulesd`:

```
2019/05/15 15:51:19 wazuh-modulesd:vulnerability-detector[16712] wm_vuln_detector.c:667 at wm_vuldet_report_agent_vulnerabilities(): DEBUG: (5456): The 'ntp' package from agent 0 is not vulnerable to CVE-2017-6451. Condition: package version (1:4.2.8p10+dfsg-3+deb9u2) less than 0:0.
2019/05/15 15:51:19 wazuh-modulesd:vulnerability-detector[16712] wm_vuln_detector.c:667 at wm_vuldet_report_agent_vulnerabilities(): DEBUG: (5456): The 'ntp' package from agent 0 is not vulnerable to CVE-2017-6452. Condition: package version (1:4.2.8p10+dfsg-3+deb9u2) less than 0:0.
2019/05/15 15:51:19 wazuh-modulesd:vulnerability-detector[16712] wm_vuln_detector.c:667 at wm_vuldet_report_agent_vulnerabilities(): DEBUG: (5456): The 'ntp' package from agent 0 is not vulnerable to CVE-2017-6455. Condition: package version (1:4.2.8p10+dfsg-3+deb9u2) less than 0:0.
2019/05/15 15:51:19 wazuh-modulesd:vulnerability-detector[16712] wm_vuln_detector.c:667 at wm_vuldet_report_agent_vulnerabilities(): DEBUG: (5456): The 'ntp' package from agent 0 is not vulnerable to CVE-2017-6458. Condition: package version (1:4.2.8p10+dfsg-3+deb9u2) less than 0:1:4.2.8p10+dfsg-1.
2019/05/15 15:51:19 wazuh-modulesd:vulnerability-detector[16712] wm_vuln_detector.c:667 at wm_vuldet_report_agent_vulnerabilities(): DEBUG: (5456): The 'ntp' package from agent 0 is not vulnerable to CVE-2017-6459. Condition: package version (1:4.2.8p10+dfsg-3+deb9u2) less than 0:0.
2019/05/15 15:51:19 wazuh-modulesd:vulnerability-detector[16712] wm_vuln_detector.c:667 at wm_vuldet_report_agent_vulnerabilities(): DEBUG: (5456): The 'ntp' package from agent 0 is not vulnerable to CVE-2017-6460. Condition: package version (1:4.2.8p10+dfsg-3+deb9u2) less than 0:1:4.2.8p10+dfsg-1.
2019/05/15 15:51:19 wazuh-modulesd:vulnerability-detector[16712] wm_vuln_detector.c:667 at wm_vuldet_report_agent_vulnerabilities(): DEBUG: (5456): The 'ntp' package from agent 0 is not vulnerable to CVE-2017-6462. Condition: package version (1:4.2.8p10+dfsg-3+deb9u2) less than 0:1:4.2.8p10+dfsg-1.
2019/05/15 15:51:19 wazuh-modulesd:vulnerability-detector[16712] wm_vuln_detector.c:667 at wm_vuldet_report_agent_vulnerabilities(): DEBUG: (5456): The 'ntp' package from agent 0 is not vulnerable to CVE-2017-6463. Condition: package version (1:4.2.8p10+dfsg-3+deb9u2) less than 0:1:4.2.8p10+dfsg-1.
2019/05/15 15:51:19 wazuh-modulesd:vulnerability-detector[16712] wm_vuln_detector.c:667 at wm_vuldet_report_agent_vulnerabilities(): DEBUG: (5456): The 'ntp' package from agent 0 is not vulnerable to CVE-2017-6464. Condition: package version (1:4.2.8p10+dfsg-3+deb9u2) less than 0:1:4.2.8p10+dfsg-1.
```

Thank you for reporting this issue! I'm going to close it, feel free to re-open it if you have any questions.

Best regards.
